src/App.tsx

```diff
@@ -1,18 +1,42 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import { Pagination } from './components/Pagination';
 import { getNumbers } from './utils';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [total] = useState(42);
```

Comment:
it shouldn't be in the state
you can even declare it outside of the component (next to `items` const)